docs/CONTRIBUTING.md

```diff
@@ -55,6 +56,59 @@ This describes the _minimum_ behavior expected from all contributors.
 
 [coc]: https://www.rust-lang.org/policies/code-of-conduct
 
+
+Within the repositories, you'll find the following directories and files,
+logically grouping common assets and providing both compiled and minified
+variations.
+
+### Repositories
+The current setup contains a three different repos, which contains different Hyperswitch components.
+
+-  [App Server][app-server-repo] - The core payments engine responsible for managing payment flows, payment unification and smart routing, App server is maintained using this repo itself.
+
+- [Web Client (SDK)][web-client-repo] - An inclusive, consistent and blended payment experience optimized for the best payment conversions
+
+- [Control center][control-centre-repo] - A dashboard for payment analytics and operations, managing payment processors or payment methods and configuring payment routing rules.
+
+[app-server-repo]: https://github.com/juspay/hyperswitch
+[web-client-repo]: https://github.com/juspay/hyperswitch-web
+[control-centre-repo]: https://github.com/juspay/hyperswitch-control-center
+
+## Files Tree Layout
```

Comment:
The top level directory structure looks different today, with some new folders and crates being present. I'm not sure how we should consider keeping this part up-to-date.

Reply:
This will be taken care in next iteration